core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/content/components/dot-edit-page-state-controller/dot-edit-page-state-controller.component.ts

```diff
@@ -166,7 +167,13 @@ we want to show the lock off so the new user can steal the lock
 
     private getStateModeOptions(pageState: DotPageRenderState): SelectItem[] {
         const items = this.variant
-            ? [...(!this.variant.variant.isOriginal ? ['edit'] : []), 'preview']
+            ? [
+                  ...(!this.variant.variant.isOriginal &&
+                  this.variant.experimentStatus === DotExperimentStatusList.DRAFT
+                      ? ['edit']
+                      : []),
+                  'preview'
+              ]
```

Comment:
Nested ternary are hard to understand... let's keep it simple, if are your friend too.

You might want to extract this condition to a function with a meaninful name.

Reply:
I agree.